routes/customers.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import APIRouter, HTTPException
+from models.customer import Customer, Customer_update
+from services.customer_service import Customer_service
+
+CustomerRouter = APIRouter()
+
+
+@CustomerRouter.put("/customers/{id}")
+def update_customer(id: str, customer_update: Customer):
+
+    customer_service = Customer_service()
```

Comment:
As dependencias da classe devem ser instanciadas uma vez apenas
Do jeito que tá, cada vez que uma função for executada, o Customer_service() vai ser instanciado

Reply:
feito!